Makefile

```diff
@@ -65,6 +65,7 @@ DIST_DIR := $(shell mktemp -d /tmp/glew.XXXXXX)/$(DIST_NAME)
 #   - use LN= on gmake command-line
 
 AR      ?= ar
+ARFLAGS.STATIC ?= cr
```

Comment:
I think that is a great idea, it makes it much clearer which flags are for what purpose.